### PR TITLE
Add getter for IncludesDeclaredInline in RecordDataSchema

### PR DESCRIPTION
**Pull request by junchuanwang**
add a getter for a private field.

### CODE REVIEW

File: data/src/main/java/com/linkedin/data/schema/RecordDataSchema.java [hunk 1]
```diff
@@ -568,6 +568,17 @@ public List<NamedDataSchema> getInclude()
     return _include;
   }
 
+  /**
+   * Get the Set of {@link NamedDataSchema} declared as inline includes.
+   * The order of declared inline includes can be obtained by calling {@link #getInclude()}.
```
Comment (mchen07) [inline data/src/main/java/com/linkedin/data/schema/RecordDataSchema.java:573]: will `_includesDeclaredInline` contain the same elements in `_include`?

Comment (junchuanwang) [reply data/src/main/java/com/linkedin/data/schema/RecordDataSchema.java:573]: yes the same element can be present in both `_includesDeclaredInline` and `_include`. (i.e. `_include` is a superset) That's why we return this `_includesDeclaredInline set` just to check the existence. 

The order was maintained in _includesDeclaredInline

Comment (junchuanwang) [reply data/src/main/java/com/linkedin/data/schema/RecordDataSchema.java:573]: e.g.
```
namespace com.linkedin.pegasus.generator.test.idl

record myincludesdep includes mybase {
  myincludesdepf: string
}

namespace com.linkedin.pegasus.generator.test.idl

typeref mytypereftoinlinercd = record typeRefInlineRecord {
  typeRefInlineRecordField: string
}

namespace com.linkedin.pegasus.generator.test.idl

record mytestforincludes includes myincludesdep,
  mytypereftoinlinercd,
  typeref inlineTyperefIncludes = record inlineTyperefIncludesRcd {inlineTyperefIncludesField: string},
   record includercd1 includes record includercd1nested {a_includercd1_includercd1nested:int} {a_includercd1:int},
   record includercd2 {a_includercd2:int} {
  mytestforincludesfield: long
}
```
`_include` and `_includesDeclaredInline` will be 
Screen Shot 2022-11-06 at 10 27 22 PM

Comment (mchen07) [reply data/src/main/java/com/linkedin/data/schema/RecordDataSchema.java:573]: thanks for detailed example.

**Review by mchen07 — APPROVED** [review]

**Pull request merged**